server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -173,10 +173,11 @@ static TransportVersion def(int id) {
     public static final TransportVersion INFERENCE_REQUEST_ADAPTIVE_RATE_LIMITING = def(8_839_0_00);
     public static final TransportVersion ML_INFERENCE_IBM_WATSONX_RERANK_ADDED = def(8_840_0_00);
     public static final TransportVersion ELASTICSEARCH_9_0 = def(9_000_0_00);
-    public static final TransportVersion REMOVE_SNAPSHOT_FAILURES = def(9_002_0_00);
-    public static final TransportVersion TRANSPORT_STATS_HANDLING_TIME_REQUIRED = def(9_003_0_00);
-    public static final TransportVersion REMOVE_DESIRED_NODE_VERSION = def(9_004_0_00);
-    public static final TransportVersion ESQL_DRIVER_TASK_DESCRIPTION = def(9_005_0_00);
+    public static final TransportVersion REMOVE_SNAPSHOT_FAILURES_90 = def(9_000_0_01);
+    public static final TransportVersion TRANSPORT_STATS_HANDLING_TIME_REQUIRED_90 = def(9_000_0_02);
+    public static final TransportVersion REMOVE_DESIRED_NODE_VERSION_90 = def(9_000_0_03);
+    public static final TransportVersion ESQL_DRIVER_TASK_DESCRIPTION_90 = def(9_000_0_04);
+
```

Comment:
For a follow up: insert a comment here that versions >= 9_001_0_00 are forbidden here? So that any auto-backport breaks, and manual backports should read it (as it would be right on the conflict line?)

Reply:
There could always be a test that failed if the transport version was ever 9_001_0_00 or above

Reply:
https://github.com/elastic/elasticsearch/pull/122016